updater/lib/dependabot/file_fetcher_command.rb

```diff
@@ -97,6 +97,7 @@ def dependency_files_for_multi_directories
         ff = with_retries { file_fetcher_for_directory(dir) }
         files = ff.files
         post_ecosystem_versions(ff) if should_record_ecosystem_versions?
+        files.each { |f| f.job_directory = dir }
```

Comment:
Could/should this be set [here](https://github.com/dependabot/dependabot-core/blob/bb7a2eb0998c29180184301611f78d38eec5a37f/common/lib/dependabot/file_fetchers/base.rb#L107) instead, using `FileFetcher#directory`?

Reply:
Yep, that works!